app/src/main/res/layout/content_card_items.xml

```diff
@@ -0,0 +1,22 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Should this just be ``content_card_item``? It doesn't have parity with its interaction feedback card counterpart.

Reply:
Made it content_card_item.